package.json

```diff
@@ -1,32 +1,55 @@
 {
-  "name": "@solana/solidity",
-  "version": "0.0.16",
-  "description": "Solana Solidity Contracts JavaScript Client",
-  "main": "dist/esm/src/index.js",
-  "types": "dist/src/index.d.ts",
-  "scripts": {
-    "lint": "rm -rf dist && tsc",
-    "prepublishOnly": "rm -rf dist && tsc --project tsconfig.release.json"
-  },
-  "license": "MIT",
-  "devDependencies": {
-    "@types/mocha": "8.0.4",
-    "@types/node": "14.14.10",
-    "eslint": "7.32.0",
-    "eslint-config-prettier": "8.3.0",
-    "eslint-plugin-flowtype": "5.9.0",
-    "eslint-plugin-import": "2.24.0",
-    "eslint-plugin-prettier": "3.4.0",
-    "expect": "26.6.2",
-    "mocha": "8.2.1",
-    "prettier": "2.3.2",
-    "ts-node": "10.2.1",
-    "typescript": "4.1.2"
-  },
-  "dependencies": {
-    "@ethersproject/abi": "5.4.1",
-    "@ethersproject/bytes": "5.4.0",
-    "@ethersproject/keccak256": "5.4.0",
-    "@solana/web3.js": "1.29.2"
-  }
+    "name": "@solana/solidity",
+    "version": "0.0.16",
+    "description": "Solana Solidity Contracts JavaScript Client",
+    "license": "Apache-2.0",
+    "engines": {
+        "node": ">= 10"
+    },
+    "publishConfig": {
+        "access": "public"
+    },
+    "main": "lib/index.js",
+    "types": "lib/index.d.ts",
```

Comment:
This should have `"type": "module"`, `"sideEffects": false`, and some other keys to enable ES modules with tree shaking.